release-python-poetry/action.yaml

```diff
@@ -0,0 +1,140 @@
+name: Reusable workflow for building and publishing python packages to TestPyPI or PyPI
+
+on:
+  workflow_call:
+    inputs:
+      scope:
```

Comment:
What is the best name for this input parameter? I used `releaseType` in my actions and you used `scope` here. I think we should agree on a fitting name across actions and workflows